src/components/Match/matchColumns.jsx

```diff
@@ -850,7 +917,10 @@ export const visionColumns = [
     ),
     tooltip: strings.tooltip_used_smoke_of_deceit,
     sortFn: true,
-    displayFn: row => (row.item_uses && row.item_uses.smoke_of_deceit) || '-',
+    displayFn: (row, column, value) => value || '-',
```

Comment:
If we didn't set the `field` property in the columnDef I think value is always undefined since I think value is row[column.field]